docs/en/quick-answers/troubleshooting/menu-entries-missing-in-launched-dcc.md

```diff
@@ -7,8 +7,22 @@ lang: en
 
 # I've launched Nuke/Maya/etc. from SG Desktop, but the Shotgun menu is missing entries
 
-{% include info title="Note" content="This document describes functionality only available if you have taken control over a Toolkit configuration. Please refer to the [Shotgun Integrations Admin Guide](https://support.shotgunsoftware.com/hc/en-us/articles/115000067493) or contact support if you do not have a more advanced configuration." %}
+The actions that show up in the Shotgun menu are contextually configured. That means the list of available actions may be different depending on the context you’re in.
+It's possible you may not be seeing the app your after because your in the wrong context.
 
-When you launch an application from [SG Desktop](https://support.shotgunsoftware.com/entries/95442947), it puts you in the Project environment by default. This environment is managed by the config file in your pipeline configuration located at config/env/project.yml. Because most of your work will probably not be done in this environment, it's not configured with many apps for you to work with.
+## Example
 
-You can use the [Shotgun File Manager](https://support.shotgunsoftware.com/hc/en-us/articles/219033088-Your-Work-Files) to select the Asset, Shot, or Task to work on which will load up the appropriate new environment for you which will then have more apps enabled with menu items in your Shotgun menu.
+When you launch an application from [Shotgun Desktop](https://support.shotgunsoftware.com/entries/95442947), it puts you in the project environment by default. This environment is managed by the config file in your pipeline configuration located at `.../config/env/project.yml`. Because most of your work will probably not be done in this environment, it's not configured with many apps for you to work with.
+
+**Default Maya Project actions:**
+
+![Shotgun Menu project actions](images/shotgun_menu_project_actions.png)
+
+You can use the [Shotgun workfiles app](https://support.shotgunsoftware.com/hc/en-us/articles/219033088-Your-Work-Files) to select the Asset, Shot, or Task to work on which will load up the appropriate new environment for you which will then have more apps enabled with menu items in your Shotgun menu.
```

Comment:
I'd capitalize Workfiles and break up this sentence a bit: 

You can use the Workfiles app](https://support.shotgunsoftware.com/hc/en-us/articles/219033088-Your-Work-Files) to select the Asset, Shot, or Task to work on. This will load up the appropriate new environment for you, which will then have more apps enabled with menu items in your Shotgun menu.

Reply:
Done!